themes/tiddlywiki/vanilla/base.tid

```diff
@@ -2285,22 +2285,29 @@ html body.tc-body.tc-single-tiddler-window {
 	padding: 4px;
 }
 
-.tc-control-panel table, .tc-control-panel table input, .tc-control-panel table textarea {
+.tc-control-panel table, 
+.tc-control-panel table input,
+.tc-control-panel table textarea {
 	width: 100%;
 }
 
+/* input type checkbox doesn't make sense with 100% width */
+.tc-control-panel table input[type=checkbox] {
+	width: inherit;
```

Comment:
This is the only addition I did make to the base.tid. 

... It's needed because of the rule above, where I did add 2 linebreaks to make it readable for humans. 

The rest is replacing spaces with tabs.

Reply:
> The rest is replacing spaces with tabs.

We've had this issue before and I thought I'd made it clear why I don't like PRs that mix functional changes with whitespace changes.